app/views/records/edit_fields/_rights.html.erb

```diff
@@ -7,7 +7,7 @@
         end ) + 
         ( content_tag :aside do
             t_uri(:description, scope: [ :rights, b.value ])
-        end )
+        end ) if !b.text.include?('3.0')
```

Comment:
Is there way to do this from the RightsService such that it only returns currently valid options from `select_options` ?